### PR TITLE
feat(styles): updated sizing utility (no tokens) 

### PR DESCRIPTION
**Pull request by myrta2302**

### CODE REVIEW

File: packages/components/cypress/e2e/logo.cy.ts [hunk 1]
```diff
@@ -49,14 +49,14 @@ describe('logo', () => {
...
-        });
-    });
+    // it('should not fit the parent height', () => {
+    //   cy.get('@logo')
+    //     .parent()
+    //     .invoke('innerHeight')
+    //     .then(parentHeight => {
+    //       cy.get('@logo').invoke('outerHeight').should('not.eq', parentHeight);
+    //     });
+    // });
```
Comment (oliverschuerch) [inline packages/components/cypress/e2e/logo.cy.ts:59]: Why is this commented out?
If we don't need this this anymore, please delete it. We can always restore it from the history if necessary.

File: packages/migrations/src/utils/constants.ts [hunk 2]
```diff
@@ -1,6 +1,8 @@
 export const breakpoints = [ 'xs', 'sm', 'rg', 'md', 'lg', 'xl', 'xxl' ];
 
 const bootstrapSizes = [ '0', '1', '2', '3', '4', '5', 'auto' ];
+
```
Comment (oliverschuerch) [inline packages/migrations/src/utils/constants.ts:4]:
```suggestion
```

**Review by oliverschuerch — CHANGES_REQUESTED** [review]

File: packages/components/cypress/e2e/logo.cy.ts (hunk 1 above)
Comment (myrta2302) [reply packages/components/cypress/e2e/logo.cy.ts:59]: yes, this was by mistake. I ll revert the file.

**Review by oliverschuerch — CHANGES_REQUESTED** [review]

File: packages/migrations/src/migrations/post/custom-select/index.ts
[the remark this answers is not in the document]
Comment (myrta2302) [reply packages/migrations/src/migrations/post/custom-select/index.ts:1]: This is just updating the mw-100 to min-w-100. Saving the file does automatically the reformat.

**Review by gfellerph — CHANGES_REQUESTED** [review]

File: packages/styles/src/utilities/_variables.scss [hunk 3]
```diff
@@ -28,6 +29,126 @@ $position-values: (
...
+    values: $post-combined-sizes,
+  ),
+  'min-width': (
+    responsive: true,
+    property: min-width,
+    class: min-w,
+    values: $post-combined-sizes,
+  ),
+  'min-height': (
+    responsive: false,
```
Comment (oliverschuerch) [inline packages/styles/src/utilities/_variables.scss:63]: Should this also be `true`?

**Review by oliverschuerch — CHANGES_REQUESTED** [review]

**Review by oliverschuerch — APPROVED** [review]

**Review by gfellerph — APPROVED** [review]

**Pull request merged**